modules/apps/commerce/commerce-frontend-js/test/dev/components/DatasetDisplay.js

```diff
@@ -221,10 +228,10 @@ const fluidDataSetDisplayProps = {
 					</>
 				);
 			},
-			icon: 'merge',
-			id: 'custom-table',
 			label: "Hey you don't know me",
```

Comment:
This looks like "random text" to test if something is working so I'm not sure, I'd suggest changing the value (I'm saying this with no context just looking at the code changes)